samples/bluetooth/hci_pwr_ctrl/README.rst

```diff
@@ -0,0 +1,35 @@
+.. _bluetooth-hci-pwr-ctrl-sample:
+
+Bluetooth: HCI Power Control
+############################
+
+Overview
+********
+
+This sample application demonstrates the dynamic Tx power control over HCI RAM
```

Comment:
Yes, you are right, actually works over USB as tested... basically this functionality is provided over the HCI regardless of the transport layer. Will modify!

Reply:
Reworked the sample [documentation](https://github.com/zephyrproject-rtos/zephyr/pull/19779/commits/5e93cb2ee03a927766c172edb1950e774b6bf16c#diff-51497db637fdff050fede888928d2b39R9) to avoid confusion.